scorpio/settings.py

```diff
@@ -163,6 +163,9 @@
     },
 }
 
+if config.ELASTICSEARCH_AUTHENTICATION:
```

Comment:
I wonder if we need this (and the ELASTICSEARCH_AUTHENTICATION) variable. In the ELASTICSEARCH setting above, I think we can do something like:

'api_key': config.get('ELASTICSEARCH_API_KEY')

That way, if that value isn't configured, the API key will be blank, and then the check when instantiating the ES client will still work.